src/icons/general/IconClockPreview.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
```

Comment:
Do we need the empty lines between imports?

Reply:
removed